### PR TITLE
chore(deps): Update PatternFly, RedHatServices and others

### PR DESCRIPTION
**Pull request by bastilian**
In addition to the usual suspects this also replaces the (deprecated) node-sass with the supported dart-sass[0].

## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [x] Input Validation
- [x] Output Encoding
- [x] Authentication and Password Management
- [x] Session Management
- [x] Access Control
- [x] Cryptographic Practices
- [x] Error Handling and Logging
- [x] Data Protection
- [x] Communication Security
- [x] System Configuration
- [x] Database Security
- [x] File Management
- [x] Memory Management
- [x] General Coding Practices

[0] https://sass-lang.com/blog/libsass-is-deprecated

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -3125,16 +3082,10 @@
...
       "dev": true
     },
-    "amdefine": {
-      "version": "1.0.1",
-      "resolved": "https://registry.npmjs.org/amdefine/-/amdefine-1.0.1.tgz",
-      "integrity": "sha1-SlKCrBZHKek2Gbz9OtFR+BfOkfU=",
-      "dev": true
-    },
     "ansi-colors": {
-      "version": "4.1.1",
```
Comment (vkrizan) [inline package-lock.json:3135]: Downgrade?

File: package-lock.json [hunk 2]
```diff
@@ -5,9 +5,9 @@
   "requires": true,
   "dependencies": {
     "@apollo/client": {
-      "version": "3.3.15",
-      "resolved": "https://registry.npmjs.org/@apollo/client/-/client-3.3.15.tgz",
-      "integrity": "sha512-/WQmNvLEZMA0mA3u+FkEPTXKzxZD/KhyO7WlbKcy3zKGrXKza83tAbNMzsitQE7DTcSc3DLEcIu1Z5Rc7PFq0Q==",
+      "version": "3.3.19",
```
Comment (vkrizan) [inline package-lock.json:8]: Are there any implications from this update? (bugfixes?, CVEs?)

File: package-lock.json [hunk 3]
```diff
@@ -13967,31 +12925,31 @@
       }
     },
     "request": {
-      "version": "2.88.2",
```
Comment (vkrizan) [inline package-lock.json:13970]: Another downgrade?

File: package-lock.json (hunk 2 above)
Comment (bastilian) [reply package-lock.json:8]: There is a changelog here: https://github.com/apollographql/apollo-client/blob/main/CHANGELOG.md

File: package-lock.json (hunk 1 above)
Comment (bastilian) [reply package-lock.json:3135]: Apparently. This is not a direct dependency of ours and I trust that `npm install` properly resolves any dependencies and required and fitting versions.

**Review by akofink — APPROVED** [review]
Thanks, @bastilian! I don't see anything out of place while testing.

**Pull request merged**